lib/cdo/pegasus_languages.rb

```diff
@@ -0,0 +1,9 @@
+require 'cdo/db'
+require 'cdo/cache_method'
+
+class PegasusLanguages < Languages
+  using CacheMethod
```

Comment:
just noting you're now `using CacheMethod` in all 3 class definitions. Could be worth quickly checking if you can remove it from 1 or 2 of them?